proposals/models.py

```diff
@@ -287,6 +289,22 @@ class Proposal(models.Model):
         null=True,
         blank=True,
     )
+    
+    avg_understood = models.BooleanField(
```

Comment:
This _technicaly_ is mixing Dutch and English, as it should be GPDR in English.

(You could defend it with the argument that the AVG is a specific Dutch law based on the generic GDPR and thus a valid name in English as well, if you feel changing it is too much work ;) )

Reply:
As I understand it they're not entirely interchangeable. Data collected in NL must comply with the AVG, and if it does, it complies with the GDPR directive as well. But the AVG implementation is more protective in certain areas than strictly necessary through the GDPR. AVG implies GDPR, but not vice versa. 

If we imagine an international researcher used to say, the French implementation of GDPR, they might break the AVG while still technically complying with GDPR. So we ask them specifically to comply with AVG. Thankfully, neither GDPR or AVG say anything about intercourse with ants.

Reply:
That's entirely true from what I believe as well, which is why I gave that defense in my comment as well :p (although I also believe the gdpr implementations of other eu states should be accepted universally... But that doesn't affect us, as that only comes into play if someone shops at a German site for example).

The thing is, our beloved bureaucrats at the government really like to be confusing. The official dutch translation of avg is actually also gdpr. (Even though the gdpr and the gdpr are indeed different). See also the English site of the AP for example.

Anyway, let's just leave it the way it is then. We have enough ants in the vending machine at work already.